content/blog/astradb-haystack-integration/index.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: blog-post
+title: Announcing the Astra DB Haystack Integration
+description: Learn how to use the new AstraDB integrations for Haystack 2.0 in your RAG pipelines.
+featured_image: thumbnail.png
+images: ["blog/astradb-haystack-integration/thumbnail.png"]
+alt_image: The logos for Haystack and Astra DB hang out on a blue background in front of some people tending to pipelines, and inexplicably a giant lightbulb.
+toc: True
+date: 2024-01-18
+last_updated:  2024-01-18
+authors:
+  - Tilde Thurium
+tags: ["Embeddings", "Haystack 2.0", "Vector Database"]
+cookbook: astradb_haystack_integrations.ipynb
+---	
+The Haystack extension family is growing so fast, it's hard to keep up! Our latest addition is the Astra DB extension by [Datastax](https://datastax.com/). It's an open source package that helps you use Astra DB as a vector database for your Haystack pipelines.
+
+Let's learn about the benefits of Astra DB and how to use it with Haystack.
+
+### Benefits of Astra DB
+
+DataStax Astra DB is a serverless vector database built on [Apache Cassandra](https://cassandra.apache.org/_/index.html). What makes Astra DB special?
+
+- **Interoperability** with Cassandra's open source ecosystem and tooling. 
+- Astra DB a **supports variety of different embedding models**. One Astra database instance can have multiple `collections` with different vector sizes. This makes it easy to test different embedding models and find the best one for your use case.
+- **It's serverless**. What does that mean for a database? You don't have to manage individual instances, or deal with cumbersome upgrading or scaling. All of that is taken care of for you behind the scenes.
+- **Enterprise scalability**. Astra DB can be deployed across the major cloud provides (AWS, GCP, or Azure) across multiple regions depending on your needs.
+- As of the time of this writing, **there's a free tier available** so you can try it without putting down a credit card.
+
+### Creating your Astra DB database
+To ensure these instructions remain up to date, we're going to point you to the Astra DB docs to explain how to create a database.
+
+Follow the first step in this [this tutorial to create a free Astra DB database](https://docs.datastax.com/en/astra-serverless/docs/manage/db/manage-create.html). Make a note of your credentials - you'll need your database ID, application token, keyspace, and database region to use the Haystack extension.
+
+[Follow these steps to create a collection](https://docs.datastax.com/en/astra/astra-db-vector/databases/manage-collections.html). Save the name of your collection since you'll need this as well. 
+
+Choose the number of dimensions that matches the [embedding model](https://haystack.deepset.ai/blog/what-is-text-vectorization-in-nlp) you plan on using. For this example we'll use a 384-dimension model, [`sentence-transformers/all-MiniLM-L6-v2`](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2).
```

Comment:
Updated links, phrasing, and put the items in a list since they need to be followed in order.

```suggestion
1. [Create a free Astra DB database](https://docs.datastax.com/en/astra/astra-db-vector/databases/create-database.html#create-vector-database). Make a note of your credentials - you'll need your database ID, application token, keyspace, and database region to use the Haystack extension.
2. Choose the number of dimensions that matches the [embedding model](https://haystack.deepset.ai/blog/what-is-text-vectorization-in-nlp) you plan on using. For this example we'll use a 384-dimension model, [`sentence-transformers/all-MiniLM-L6-v2`](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2).
3. [Create a collection](https://docs.datastax.com/en/astra/astra-db-vector/databases/manage-collections.html#create-collection) with the same number of dimensions as your embedding model. Save the name of your collection since you'll need this as well. 
```